pkg/controller/bootstrap/bootstrapsigner.go

```diff
@@ -200,8 +202,7 @@ func (e *Signer) signConfigMap(ctx context.Context) {
 	// First capture the config we are signing
 	content, ok := newCM.Data[bootstrapapi.KubeConfigKey]
 	if !ok {
-		klog.V(3).Infof("No %s key in %s/%s ConfigMap", bootstrapapi.KubeConfigKey, origCM.Namespace, origCM.Name)
```

Comment:
providing logs with --v=3 or higher would have helped us better understand why this happens:
https://github.com/kubernetes/kubernetes/issues/109233#issuecomment-1090355503

Reply:
The original code is --v=3, i don't think it need changed